doc/contributing.rst

```diff
@@ -589,4 +589,4 @@ use `lib/std/collections/foo.nim`, not `lib/pure/collections/foo.nim`.
 `std/sums.nim` instead of `std/sum.nim`. In particular, this reduces chances of conflicts
 between module name and the symbols it defines. Furthermore, is should use `snake_case`
```

Comment:
replace "is" with "module names"

Reply:
Ah, nice catch. Not sure how I missed that. Fixed.